NitroxPatcher/Patches/Dynamic/Drillable_OnDrill_Patch.cs

```diff
@@ -0,0 +1,43 @@
+using System.Reflection;
+using Nitrox.Model.DataStructures;
+using Nitrox.Model.Subnautica.DataStructures.GameLogic;
+using Nitrox.Model.Subnautica.DataStructures.GameLogic.Entities;
+using Nitrox.Model.Subnautica.DataStructures.GameLogic.Entities.Metadata;
+using Nitrox.Model.Subnautica.Packets;
+using NitroxClient.Communication.Abstract;
+using NitroxClient.GameLogic;
+using NitroxClient.GameLogic.Spawning.Metadata;
+using NitroxClient.MonoBehaviours;
+using UnityEngine;
+
+namespace NitroxPatcher.Patches.Dynamic;
+
+public sealed partial class Drillable_OnDrill_Patch : NitroxPatch, IDynamicPatch
+{
+    public static readonly MethodInfo TARGET_METHOD = Reflect.Method((Drillable t) => t.OnDrill(default, default, out Reflect.Ref<GameObject>.Field));
+
+    public static void Postfix(Drillable __instance)
+    {
+        if (__instance.TryGetNitroxId(out NitroxId id))
+        {
+            Resolve<Entities>().EntityMetadataChanged(__instance, id);
+            return;
+        }
+
+        // For some reason the drillable ion cube deposit in the primary containment facility
+        // is not automatically tagged as an entity. We spawn it here as a workaround.
```

Comment:
Would at least add a TODO: that this should be moved to a spawn bootstrapper inside the server